opty/direct_collocation.py

```diff
@@ -727,6 +729,77 @@ def __init__(self, equations_of_motion, state_symbols,
             If True, STDOUT and STDERR of the Cython compilation call will be
             shown.
 
+        Attributes
+        ----------
```

Comment:
```suggestion
        ----------
        ==========
```

match heading level of parameters